### PR TITLE
New Cloud Release notes (2022-09-19-ReleaseNotes)

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+![Phone-auth](images/Phone-auth.png)
+
+As your Umbraco profile is used to log in to the backoffice for your projects, these are all now protected with 2FA.
+
+See the [2-factor authentication on Umbraco Cloud documentation](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/) for details. 
+
+## US Region Hosting
+
+Until now, Umbraco Cloud has been exclusively hosted in the West Europe region on Azure. Including **East US** as an option and being able to offer regional hosting to agencies, partners, and end clients that are located or work in the US is a big milestone and enabler for Umbraco Cloud.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:29]:
```suggestion
Until now, Umbraco Cloud has been exclusively hosted in the West Europe region on Azure. Including **East US** as an option and offering regional hosting to agencies, partners, and end clients located or working in the US is a big milestone and enabler for Umbraco Cloud.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+You can easily select which region you would like to host your Umbraco Cloud project while creating a new Cloud project.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:31]:
```suggestion
You can select which region you would like to host your Umbraco Cloud project while creating a new Cloud project.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+![2FA-Registration](images/2FA-Registration.gif)
+
+As excited as we are about making Umbraco Cloud available in the US, it is just as exciting to have the ability to expand to **new regions in the future**. If you have specific requests regarding the next region you want Umbraco Cloud to expand to, you’re welcome to let us know at product@umbraco.com.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:35]:
```suggestion
As excited as we are about making Umbraco Cloud available in the US, it is exciting to have the ability to expand to **new regions in the future**. If you have specific requests regarding the next region you want Umbraco Cloud to expand to, you’re welcome to let us know at product@umbraco.com.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+See the blog article [Umbraco Cloud expands with US regional hosting](https://umbraco.com/blog/umbraco-cloud-expands-with-us-regional-hosting/) for more information.
+
+## Projects Dashboard
+
+Psst…have you tried the new Projects Dashboard yet? The first official version of the dashboard has been released. Currently, you can find a link in the projects overview, but soon the Projects Dashboard will be the default landing page after logging into Umbraco Cloud Portal.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:41]:
```suggestion
Have you tried the new Projects Dashboard yet? The first official version of the dashboard has been released. Currently, you can find a link in the project's overview, but soon the Projects Dashboard will be the default landing page after logging into Umbraco Cloud Portal.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+By pressing the button “Edit groups” you are able to create project groups and with simple drag-drop functionality organize your cloud and Heartcore projects in project groups.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:43]: As per the gif looks like sorting the projects more than creating project groups.

```suggestion
By clicking on “Edit groups”, you can sort projects into groups with the drag-drop functionality to organize your Cloud and Heartcore projects.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+After saving your projects, you will be able to choose between a detailed view with relevant information for your projects such as environments, product versions, and bandwidth/media storage usage, or a more compact view with less information (you decide by using the “Settings” option).
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:45]:
```suggestion
After saving your projects, you can choose between a detailed view with relevant information for your projects such as environments, product versions, and bandwidth/media storage usage, or a more compact view with less information. You can decide by using the “Settings” option.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
+# Release Notes, September 19, 2022
+
+_2FA authentication - US Region Hosting - Projects Dashboard_
+
+## Key Takeaways
+
+- **2FA authentication** - You can now make your Umbraco Cloud account even more secure by enabling 2-factor authentication (2FA). Select between email, phone, or app authentication.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:7]:
```suggestion
- **2FA authentication** - You can now secure your Umbraco Cloud account by enabling 2-factor authentication (2FA). Select between email, phone, or app authentication.
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+See the [2-factor authentication on Umbraco Cloud documentation](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/) for details. 
+
+## US Region Hosting
+
+Until now, Umbraco Cloud has been exclusively hosted in the West Europe region on Azure. Including **East US** as an option and being able to offer regional hosting to agencies, partners, and end clients that are located or work in the US is a big milestone and enabler for Umbraco Cloud.
+
+You can easily select which region you would like to host your Umbraco Cloud project while creating a new Cloud project.
+
+![2FA-Registration](images/2FA-Registration.gif)
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:33]: Looks like the wrong gif.
```suggestion
![USRegionRegistration](images/USRegionRegistration.gif)
```

File: Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+Psst…have you tried the new Projects Dashboard yet? The first official version of the dashboard has been released. Currently, you can find a link in the projects overview, but soon the Projects Dashboard will be the default landing page after logging into Umbraco Cloud Portal.
+
+By pressing the button “Edit groups” you are able to create project groups and with simple drag-drop functionality organize your cloud and Heartcore projects in project groups.
+
+After saving your projects, you will be able to choose between a detailed view with relevant information for your projects such as environments, product versions, and bandwidth/media storage usage, or a more compact view with less information (you decide by using the “Settings” option).
+
+![ProjectsDashboardV1](images/ProjectsDashboardV1.gif)
+
+You can try out the Projects Dashboard on https://www.s1.umbraco.io/projectsdashboard and start grouping the Cloud projects of your preference.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-09-19-ReleaseNotes.md:49]:
```suggestion
You can try out the Projects Dashboard on https://www.s1.umbraco.io/projectsdashboard and start grouping the Cloud projects as per your preference.
```

**Comment by sajumb** [comment]
Awesome! Thanks a lot, Esha! All changes approved :-)

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @sajumb 🙌 Merging it in.

**Pull request merged**